models/payment.go

```diff
@@ -50,3 +51,19 @@ type Data struct {
 type Filing struct {
 	Description string `json:"description"`
 }
+
+// CostResource contains the details of an individual Cost Resource
+type CostResource struct {
```

Comment:
Looking at the documentation the CostResource can also contain a "redirect_uri" which might need to be added in here?

Reply:
`redirect_uri` added to Payment resource